.gitlab.mk

```diff
@@ -115,7 +115,7 @@ package: deploy_prepare
 		export VERSION="$$(echo $(GIT_DESCRIBE) | sed 's/-\([0-9]\+\)-g[0-9a-f]\+$$/.\1/' | sed 's/-/~/').dev"; \
 	fi;                                                                                                             \
 	echo VERSION=$$VERSION;                                                                                         \
-	PACKPACK_EXTRA_DOCKER_RUN_PARAMS="--network=host ${PACKPACK_EXTRA_DOCKER_RUN_PARAMS}"                           \
+	PACKPACK_EXTRA_DOCKER_RUN_PARAMS="--network=host --volume /tmp/t:/tmp/t ${PACKPACK_EXTRA_DOCKER_RUN_PARAMS}"    \
```

Comment:
Is it related to artifacts?

Reply:
Yep, package testing runs in containers, so we need to create a volume to be able to archive logs in the case of a failure.